commons-library/src/main/java/com/afh/skytouch/commons/dto/GenericProduct.java

```diff
@@ -26,4 +34,18 @@
         id = UUID.randomUUID();
         creationDate = Instant.now();
     }
+
+    @Override
+    public boolean equals(Object object){
```

Comment:
When you require to overwrite this methods I recommend to let the IDE to create them for you. For example in this case your method does not validate null values... give it a try

Reply:
Thanks I will try it!